vertexai/function-calling/functioncalling.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// function-calling shows an example of OpenAPI Function Call
+//
+// This Function Call involves 3 messages:
```

Comment:
Lower case "function call" unless this is a proper noun.

Reply:
If "Function Call" is a proper noun -- that is, if it is a unique feature of Vertex -- provide a link to the documentation on cloud.google.com.

Reply:
According to [this doc page](https://cloud.google.com/vertex-ai/docs/generative-ai/multimodal/function-calling), lower case should be prefered.